### DISCUSSION TITLE
ARM64 supprt?

### DISCUSSION THREAD

**Discussion by nixyhustle** [tree-sitter/tree-sitter#1886] [Q&A]
Hello world

Just curious if there is ARM64 support yet, or perhaps any news that it may be in the works? 

Thanks!

**Comment by ahlinc** [comment]
Could you provide more details about a platform (compiler errors, etc.) about what don't work there?

**Reply by nixyhustle** [reply]
Sorry for the late reply, it was an error on my part. I appreciate your response and time.
